.gitignore

```diff
@@ -18,3 +18,4 @@ yarn.lock
 .idea
 .vscode
 pre-commit.sh
+tsdef/twilio-video-tests.js
```

Comment:
This shouldn't be necessary if the JS isn't emitted by using `--noEmit` with `tsc`.